File: app/src/main/java/com/zhai/mvpdemo/AppComponent.java
package com.zhai.mvpdemo;

import android.app.Application;

import javax.inject.Singleton;

import dagger.Component;

/**
 * Created by zhai on 16/6/20.
 */
@Singleton
@Component(modules = AppModule.class)
public interface AppComponent {
    void inject(Application application);
}

File: app/src/main/java/com/zhai/mvpdemo/AppComponent.kt
package com.zhai.mvpdemo

import android.app.Application
import dagger.Component
import javax.inject.Singleton

/**
 * Created by zhai on 16/6/20.
 */
@Singleton
@Component(modules = arrayOf(AppModule::class))
interface AppComponent {
    fun inject(application: Application)
}
